test/infrastructure/docker/docker/machine.go

```diff
@@ -238,12 +238,13 @@ func (m *Machine) ExecBootstrap(ctx context.Context, data string) error {
 	for _, command := range commands {
 		cmd := m.container.Commander.Command(command.Cmd, command.Args...)
 		cmd.SetStderr(&out)
+		cmd.SetStdout(&out)
 		if command.Stdin != "" {
 			cmd.SetStdin(strings.NewReader(command.Stdin))
 		}
 		err := cmd.Run(ctx)
 		if err != nil {
-			m.log.Info("Failed running command", "command", command, "stderr", out.String())
+			m.log.Info("Failed running command", "command", command, "stderr", out.String(), "cloud init", cloudConfig)
```

Comment:
Let's maybe use separate buffers for stdout & stderr?

Reply:
Done